core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -545,7 +545,6 @@ public synchronized void gainPrimacy() {
     mAsyncJournalWriter
         .set(new AsyncJournalWriter(mRaftJournalWriter, () -> getJournalSinks(null)));
     mTransferLeaderAllowed.set(true);
-    super.registerMetrics();
```

Comment:
The way sequence number works differ for UFS journal vs embedded journal.

For UFS journal, each XXXMaster has a separate sequence which can be totally different. It can look like BlockMaster has 1L and FileSystemMaster has 100L. For EJ, all XXXMaster share a global `mStateMachine.getLastAppliedSequenceNumber()`. So I think instead of either reusing the old metrics or removing those metrics, we can do separate handling for UFS journal and EJ.

For UfsJournalSystem we can keep the old implementation from `AbstractJournalSystem.registerMetrics()`, and for RaftJournalSystem you can declare ONE different metric which contains the global sequence number. @elega WDYT?